sdk/eventgrid/azure-eventgrid/swagger/README.PYTHON_T2.md

```diff
@@ -0,0 +1,19 @@
+# Azure EventGrid Client Generation for Python
+
+> see https://aka.ms/autorest
+
+### Configuration
+
+```yaml
+title: EventGridPublisherClient
+description: EventGrid Python Publisher Client
+generated-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../azure/eventgrid/_generated
+source-code-folder-path: ./azure/eventgrid/_generated
+input-file: 
+    - https://raw.githubusercontent.com/t-swpill/azure-rest-api-specs/add-cloud-event-publish-to-event-grid/specification/eventgrid/data-plane/Microsoft.EventGrid/stable/2018-01-01/EventGrid.json
```

Comment:
why is this file checked into a fork?

Reply:
I was generating the Publisher client based on this, but needed to change the spec from the original version. Matt also has his version here: https://github.com/ellismg/azure-rest-api-specs/blob/ellismg/add-cloud-event-publish-to-event-grid/specification/eventgrid/data-plane/Microsoft.EventGrid/stable/2018-01-01/EventGrid.json. I changed my spec to be consistent with Matt's , except for line 230:
```"data_base64": {
          "description": "Event data specific to the event type, encoded as a base64 string.",
          "type": "string",
          "format": "byte"
        }
```
This addition of `"format": "byte"` was per Laurent's direction.